lib/install/management/appliance.go

```diff
@@ -146,7 +149,7 @@ func (d *Dispatcher) getName(vm *vm.VirtualMachine) string {
 	return name
 }
 
-func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine, force bool) error {
+func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec, force bool) error {
```

Comment:
Still depends on how would you implement this logic, but ideally it does not impact container vm deletion. You might add a new method for appliance vm deletion.